tests/manage/mcg/test_write_to_bucket.py

```diff
@@ -360,3 +364,39 @@ def test_write_to_bucket_rbd_cephfs(
                     awscli_pod_session, target_dir, full_object_path, mcg_obj
                 )
             )
+
+    @tier2
+    @bugzilla("2054074")
```

Comment:
add skip if version if not supported in versions others than fixed in version of bz

Reply:
ack